{
    "id": "CVE-2021-30005",
    "product": "n/a",
    "version": "n/a",
    "vulnerability": [
        "n/a"
    ],
    "description": "In JetBrains PyCharm before 2020.3.4, local code execution was possible because of insufficient checks when getting the project from VCS.",
    "poc": {
        "reference": [
            "No PoCs from references."
        ],
        "github": [
            "https://github.com/ARPSyndicate/cvemon",
            "https://github.com/SYRTI/POC_to_review",
            "https://github.com/WhooAmii/POC_to_review",
            "https://github.com/atorralba/CVE-2021-30005-POC",
            "https://github.com/nomi-sec/PoC-in-GitHub",
            "https://github.com/soosmile/POC",
            "https://github.com/trhacknon/Pocingit",
            "https://github.com/zecool/cve"
        ]
    }
}